Makefile

```diff
@@ -0,0 +1,76 @@
+CHS_ENV_HOME  ?= $(HOME)/.chs_env
+TESTS         ?= ./...
+
+bin           := payment-reconciliation-consumer
+test_path     := ./test
+chs_envs      := $(CHS_ENV_HOME)/global_env $(CHS_ENV_HOME)/payment-reconciliation-consumer/env
+source_env    := for chs_env in $(chs_envs); do test -f $$chs_env && . $$chs_env; done
+xunit_output  := test.xml
+lint_output   := lint.txt
+
+commit        := $(shell git rev-parse --short HEAD)
+tag           := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version       := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
+
+.PHONY: all
+all: build
+
+.PHONY: fmt
+fmt:
+	go fmt ./...
+
+.PHONY: deps
+deps:
+	go get github.com/mitchellh/gox
+	go get github.com/companieshouse/$(bin)
+
+.PHONY: build
+build: deps fmt $(bin)
+
+$(bin):
+	go build -o ./$(bin)
+
+.PHONY: test-deps
+test-deps: deps
+	go get github.com/smartystreets/goconvey
+	go get github.com/smartystreets/assertions
+
+.PHONY: test
+test: test-unit test-integration
+
+.PHONY: test-unit
+test-unit: test-deps
+	@set -a; go test $(TESTS) -run 'Unit'
+
+.PHONY: test-integration
```

Comment:
Is this required? We won't be running integration tests against this application during the build

Reply:
But shouldn't we run integration test as this application communicates with the payments.api ?

Reply:
We won't be able to until we write an API automation test suite

Reply:
what about something like this : https://github.com/companieshouse/chs-streaming-api/blob/develop/integrationtesting/stream_integration_test.go

Reply:
Point taken, ignore this then 👍